app/models/preserved_copy.rb

```diff
@@ -3,13 +3,21 @@
 class PreservedCopy < ApplicationRecord
   belongs_to :preserved_object
   belongs_to :endpoint
-  belongs_to :status
 
   validates :preserved_object, presence: true
   validates :endpoint, presence: true
   validates :version, presence: true
-  validates :status, presence: true
+
   # NOTE: size here is approximate and not used for fixity checking
   validates :size, numericality: { only_integer: true, greater_than: 0 }, allow_nil: true
-
+  enum status: {
+    ok: 0,
+    invalid_moab: 1,
+    invalid_checksum: 2,
+    online_moab_not_found: 3,
+    expected_version_not_found_online: 4,
+    fixity_check_failed: 5
+  }
+  DEFAULT_STATUS = statuses[:ok].freeze
```

Comment:
lovin' this 'DEFAULT_STATUS' constant

did the Settings.yml.  change to remove status stuff?   And/or github issues to update shared_configs?

Reply:
shared_configs doesn't have status stuff...when i checked our two branches